src/Compilers/CSharp/Portable/Lowering/MethodToClassRewriter.cs

```diff
@@ -672,12 +672,12 @@ internal override void AddSynthesizedAttributes(PEModuleBuilder moduleBuilder, r
                 AddSynthesizedAttribute(ref attributes, this.DeclaringCompilation.TrySynthesizeAttribute(WellKnownMember.System_Diagnostics_DebuggerHiddenAttribute__ctor));
             }
 
-            internal override TypeSymbol IteratorElementType
+            internal override TypeSymbolWithAnnotations IteratorElementType
             {
                 get
                 {
                     // BaseMethodWrapperSymbol should not be rewritten by the IteratorRewriter
-                    return null;
+                    return TypeSymbolWithAnnotations.Create(null);
```

Comment:
>TypeSymbolWithAnnotations.Create(null) [](start = 27, length = 38)

`default`

Reply:
Rather that `default` should we add a `static` property with a decent name: Missing, Null, etc ...